CHANGELOG_iOS.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## [1.71.125](https://github.com/brave/brave-browser/releases/tag/v1.71.125)
+
+### Web3
+
+ - Added support for resolving new UD TLDs. ([#40543](https://github.com/brave/brave-browser/issues/40543))
+ - Fixed creating send transaction when sending any non-compressed Solana NFT. ([#40424](https://github.com/brave/brave-browser/issues/40424))
+
+### General
+
+ - Implemented local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```

Comment:
Should add some context re: where we added it. We should mention Leo to give more context. Something as per the following:

```
  - Implemented local vector search for the selection of key text segments of page and video context in Brave Leo. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```

OR

```
 - Implemented Brave Leo local vector search for the selection of key text segments of page and video context. ([#36801](https://github.com/brave/brave-browser/issues/36801))
```